app/models/concerns/international_phone_number_validator.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# Validates that the international phone number is valid
+class InternationalPhoneNumberValidator < ActiveModel::Validator
```

Comment:
I think this validator could instead follow the pattern used by the `EmailValidator`, so that it only applied to one field, instead of the entire record. And then when we validate, instead of placing the error on `:base`, we can place it on `international_telephone`, so that an error message can be logged indicating the path in FHIR of the telephone.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1069/commits/c44c099e7284dd1335eed9023d81f655f7a29685